scripts/kind-with-registry.sh

```diff
@@ -36,7 +36,7 @@ kind_network='kind'
 reg_name='kind-registry'
 reg_port='5000'
 case "${kind_version}" in
-  "kind v0.7."* | "kind v0.6."* | "kind v0.5."*)
+  "kind v0.10."* | "kind v0.9."*)
```

Comment:
How future-proof is this, would it need to be updated when kind v0.11.0 comes out?

Reply:
yeah i think we should set it without the if condition

Reply:
Found this kind PR https://github.com/kubernetes-sigs/kind/pull/1538/files. The `KIND_EXPERIMENTAL_DOCKER_NETWORK` environment variable was added in kind `v0.8.0`.

So I believe here we should check if kind version is >= `0.8.0`. Then if we got kind >= `0.8.0`, and `KIND_EXPERIMENTAL_DOCKER_NETWORK` is set, use whatever network name is set in `KIND_EXPERIMENTAL_DOCKER_NETWORK` environment variable (`bridge` here), otherwise use default network name (`kind`).

Reply:
Or we set kind `v0.8.0` as a minimum kind version (should be fine IMHO since `v0.10.0` is the latest), and just use `KIND_EXPERIMENTAL_DOCKER_NETWORK` for network name if it is set, otherwise fallback to default network name `kind`.